cinema/views.py

```diff
@@ -1,10 +1,21 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
 from rest_framework.pagination import PageNumberPagination
-
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession, Order
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+
+from cinema_service.permissions import IsAdminOrIfAuthenticatedReadOnly
+
```

Comment:
There is no need to add a blank line between imports that belong to the same import group according to PEP-8, remove it, please